### PR TITLE
Marian renovate dropdown and remove line from abbr

### PR DESCRIPTION
**Pull request by mmlumba**
References issue #13 
screen shot 2017-01-24 at 1 57 21 pm

And issue #21 
screen shot 2017-01-24 at 1 46 17 pm

PTAL @KHarshawat @lakohl 

Issue #13 will also require adding the following to `app/view/layouts/application.html.erb` on the last menu bullet point to _all_ Caseflow apps: `<div class="dropdown-border"></div>`

### CODE REVIEW

**Comment by mmlumba** [comment]
@astewarttistatech @kierachell Instructions for testing:
Find the line `gem "caseflow"`within the `Gemfile` for any app (certification, efolder, feedback) and modify the line to look like the following:
`gem "caseflow", git: "https://github.com/department-of-veterans-affairs/caseflow-commons", ref: "0ffd3c7"`
Run `bundle install`
View the app in Dev and verify that the dropdown matches this spec:
image
Also verify that there is no dotted underline in the footer (see above screenshot).
Revert the `gem "caseflow"` line back to its original state

**Comment by lakohl** [comment]
@mmlumba  I checked the dropdown in the dev versions of eX, Feedback, and Certification and I'm not seeing your changes.

**Comment by lakohl** [comment]
@mmlumba  I checked the dropdown in the dev versions of eX, Feedback, and Certification and I'm not seeing your changes.

**Comment by mmlumba** [comment]
@lakohl The changes would only be visible after I sync the apps with the latest changes in caseflow-commons. I would like your approval to see if my screenshots look good with the specs listed from issues #13 and #21

**Comment by lakohl** [comment]
Looks much better! 
No dots showing under the heart or Digital Service.
Dropdown selection looks good. Need to adjust the padding to be consistent 20px around the dropdown box and above Sign In. The line above Sign In should not span the entire box, but rather should start under the dropdown text and stop before the edge of the box with equal spacing on either side.

**Comment by mmlumba** [comment]
Updated dropdown:
screen shot 2017-01-27 at 11 04 03 am

File: app/assets/stylesheets/caseflow_main.scss [hunk 1]
```diff
@@ -255,24 +265,40 @@ dd {
...
+      background-color: $color-gray;
+      color: $color-white;
+    }
+  }
+
+  .dropdown-border {
+    width: 175px;
+    height: 1px;
+    background-color: $color-gray-lighter;
+    margin: 0 20px;
```
Comment (KHarshawat) [inline app/assets/stylesheets/caseflow_main.scss]: @mmlumba we probably need to make this margin a little bigger. Come find me and I'll work with you to tweak it in the dom.

**Pull request merged**